concepts/closures/about.md

```diff
@@ -1,35 +1,40 @@
 # About
 
-[Closures][closures] in Swift are self-contained blocks of code that can be passed parameters to trigger their computation and return values. Closures also capture values from their environment and use them in their computations. As they are self contained, they may be passed around in a program like other values or assigned to constants and variables.
+[Closures][closures] in Swift are self-contained blocks of code that can be passed parameters to trigger their computation and return values
+Closures also capture values from their environment and use them in their computations.
+As they are self contained, they may be passed around in a program like other values or assigned to constants and variables.
 
-Closures may sound a lot like Swift functions; they do, and for good reason. Functions in swift are just special cases of closures which are required to have a name and are defined using a slightly different syntax.
+Closures may sound a lot like Swift functions; they do, and for good reason.
```

Comment:
They do ... sound like functions? Does that just repeat the first half of the sentence?

Reply:
I guess, I didn't write it originally